docs/source/general/contributing.rst

```diff
@@ -16,6 +16,44 @@ If you intend to contribute in any way please keep the following guidelines in m
 8. Document your changes
 9. Send pull request
 
+
+Virtual Environment
+-------------------
+
+Please use a virtual environment while developing your feature, it keeps everybody on the same page and it helps reproducing bugs and resolving problems.
+
+.. code-block:: shell
+
+    virtualenv env
+    source env/bin/activate
+    python setup.py install
```

Comment:
`python setup.py develop`

It installs the current source code without moving it to the env directory, it's very handy for development